CHANGES.rst

```diff
@@ -55,6 +55,8 @@ New Features
 
   - Header allows a dictionary-like cards argument during creation. [#4663]
 
+  - New function ``convenience.table_to_hdu``. [#4778]
```

Comment:
This could be a bit more descriptive, e.g. "New function ``convenience.table_to_hdu`` to allow creating a FITS HDU object directly from an astropy ``Table``".